src/scss/grommet-core/_objects.tabs.scss

```diff
@@ -5,16 +5,33 @@
   padding: 0;
   display: flex;
   flex-wrap: wrap;
-  justify-content: center;
   align-items: center;
   list-style: none;
   border-bottom: 1px solid $border-color;
 
   &__content {
-    padding: $inuit-base-spacing-unit;
-
     @include media-query(palm) {
       padding: $inuit-base-spacing-unit 0;
     }
   }
+
+  &--justify-center {
+    justify-content: center;
+  }
+
+  &--justify-start {
+    justify-content: flex-start;
+
+    @include media-query(portable) {
```

Comment:
I don't think either of them look great. Instead, I'm inclined to convert them to direction="column" and just stack them, and add a responsive={false} to Tabs to turn off this behavior. That allows callers that have a few tabs with small names to still work horizontally.